specification/communication/Communication.Messages/models.tsp

```diff
@@ -0,0 +1,364 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+@versioned(Azure.Communication.MessagesService.Versions)
+namespace Azure.Communication.MessagesService;
+
+@doc("The type of message. Supports text, image, template.")
+enum CommunicationMessageKind {
+  @doc("Text message type.")
```

Comment:
Generally, we recommend using `/** docstring */` syntax over `@doc` because you will get better IDE syntax highlighting.

Reply:
Thanks for the suggestion, this is how it is being written in other services too: https://github.com/Azure/azure-rest-api-specs/blob/6cecaf43ab9c1104e683aea3b185b88f81be2750/specification/communication/Communication.JobRouter/models.tsp
Is there a strong preference on changing to `/** docstring */`?

Reply:
It's mostly for your benefit. I don't think the Stewardship board or archboard reviewers even see these (by design).

Reply:
I often turn them on when reviewing to help explain what I'm reviewing. Not required to change, but I will admit that `/** */` comments syntax highlight better. Still, I don't see a need to change this. Maybe next service.